src/platforms/node/common/enriching-events/scopes/index.mdx

```diff
@@ -69,11 +79,7 @@ In the following example we use <PlatformIdentifier name="with-scope" /> to atta
 
 <PlatformContent includePath="enriching-events/scopes/with-scope" />
 
-While this example looks similar to <PlatformIdentifier name="configure-scope" />, it is actually very different.
-Calls to <PlatformIdentifier name="configure-scope" /> change the current active scope; all successive calls to <PlatformIdentifier name="configure-scope" /> will maintain previously set changes unless they are explicitly unset.
-
-On the other hand, <PlatformIdentifier name="with-scope" /> creates a clone of the current scope, and the changes
-made will stay isolated within the <PlatformIdentifier name="with-scope" /> callback function. This allows you to
+The scope inside of the `withScope()` callback is only valid inside of the callback - once the callback ended, the scope will be removed and no longer applied. The inner scope is only applied to events that are captured inside of the callback. `withScope()` will clone (or fork) the current scope, so that the current scope is not modified. This allows you to
```

Comment:
l: Maybe that's a little clearer, given that "inner scope" isn't mentioned before and people could mistake it for something else

```suggestion
The scope inside of the `withScope()` callback is only valid inside of the callback - once the callback ended, the scope will be removed and no longer applied. The callback scope is only applied to events that are captured inside of the callback. `withScope()` will clone (or fork) the current scope, so that the current scope is not modified. This allows you to
```